training/model_management/src/azureml/model/mgmt/processors/hftransformers/vision/requirements.txt

```diff
@@ -1,3 +1 @@
---extra-index-url https://automlcesdkdataresources.blob.core.windows.net/finetuning-models/eval_metrics_whls/17_03_latest_eval_wheel/azureml_evaluate_mlflow-0.1.0.86769376-py3-none-any.whl
 datasets==2.3.2
```

Comment:
Can we check if this can be added in model conda.yaml? It will be great to get rid of any additional dep mgmt.